ingest/bin/genbank-url

```diff
@@ -12,54 +12,85 @@ and observing the network activity at
     https://www.ncbi.nlm.nih.gov/labs/virus/vssi/#/virus?SeqType_s=Nucleotide&VirusLineage_ss=Monkeypox%20virus,%20taxid:10244
 """
 from urllib.parse import urlencode
+import argparse
 
-endpoint = "https://www.ncbi.nlm.nih.gov/genomes/VirusVariation/vvsearch2/"
-params = {
-    # Search criteria
-    'fq': [
-        '{!tag=SeqType_s}SeqType_s:("Nucleotide")', # Nucleotide sequences (as opposed to protein)
-        'VirusLineageId_ss:(10244)',                # NCBI Taxon id for Monkeypox
-    ],
+def parse_args():
+    """
+    Define parse args
+    """
+    parser = argparse.ArgumentParser(
+        description="Given an NCBI taxon ID, generate URL to download "
+        "all viral sequences and their curated metadata from GenBank via NCBI Virus."
+    )
+    parser.add_argument(
+        "--taxonid",
+        help="NCBI Taxon ID.",
+        required=True
+    )
```

Comment:
_non-blocking_

`--taxon-id` would be a more canonical option name.

Maybe default to 10244 so that any existing callers outside of this repo don't have to change?

Reply:
Agree! Basically tried to convert all parameter names to "ncbi-taxon-id" and set default to 10244. 

Thanks for helpful suggestions and let me know if you catch anything else!